snippets/spring-data/analytics.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Create Analytics Query": {
+        "prefix": ["@cbsdanq"],
+        "body": ["AnalyticsQuery query = new AnalyticsQuery();","${1:CouchbaseTemplate}.findByAnalytics(${2:Model}.class).matching(query);"],
+        "description": "Use SQL++ to Query the Analytics Service on the Couchbase Cluster."
+    },
+    "Analytics Query With Parameters": {
+        "prefix": ["@cbsdanparamq"],
+        "body": ["AnalyticsOptions opts = AnalyticsOptions.analyticsOptions().parameters(${1:parameters});","AnalyticsQuery query = new AnalyticsQuery();","${2:CouchbaseTemplate}.findByAnalytics(${3:Model}.class).withOptions(opts).matching(query);"],
+        "description": "Use SQL++ to Query the Analytics Service on the Couchbase Cluster, this uses parameters."
+    },
+    "Analytics Query Example": {
+        "prefix": ["@cbsdanex"],
+        "body": [],
+        "description": "Example code that shows connecting to a cluster and running an analytics query on it."
+    },
+    "Analytics Query Import": {
+        "prefix": ["@cbsdanimp"],
+        "body": [],
```

Comment:
Am I missing something with all these snippets with empty bodies?  It looks like they're just not going to do anything?  Should they maybe be removed?